index.html

```diff
@@ -128,148 +136,233 @@
   <div class="container">
     <h2>Registration for the August 2022 DESC Meeting in Chicago</h2>
     <h4>Monday - Friday, August 1-5, 2022</h4>
-    There is an in-person registration fee for this meeting.  Remote
+    There is an in-person registration fee for this meeting. Remote
     participation is free.
     <br>
-    Please join the LSSTC Slack channel <a href="https://lsstc.slack.com/archives/C90AMAG80" target="_blank">#desc-collab-meeting</a> to keep updated on chats, discussions, and announcements during the meeting.
+    Please join the LSSTC Slack channel <a href="https://lsstc.slack.com/archives/C90AMAG80"
+      target="_blank">#desc-collab-meeting</a> to keep updated on chats, discussions, and announcements during the
+    meeting.
     <br>
-    You can find the the details of the schedule on this <a href="https://confluence.slac.stanford.edu/x/agvUEw" target="_blank">confluence page</a>.
+    You can find the the details of the schedule on this <a href="https://confluence.slac.stanford.edu/x/agvUEw"
```

Comment:
This currently brings up the home page for the Feb 2022  Meeting.  
URL for new home page is 
https://confluence.slac.stanford.edu/x/CZh8F

URL for new (currently empty) agenda page is
https://confluence.slac.stanford.edu/x/ooXDF